src/sync/installation.ts

```diff
@@ -108,10 +110,10 @@ export const updateTaskStatusAndContinue = async (
 			}
 		}
 
-		statsd.increment(metricTaskStatus.complete, [`type:${task.task}`, `gitHubProduct:${gitHubProduct}`]);
+		statsd.increment(metricTaskStatus.complete, duration, [`type:${task.task}`, `gitHubProduct:${gitHubProduct}`]);
```

Comment:
Shouldn't it be a separate metric? If I understand the signature of this method correctly, you are going to add this number of milliseconds, which probably is not something we want? (I'd expect to use histogram for duration...)

Reply:
Yeah I agree. We use histogram to show time, in 'buckets'. Example https://github.com/atlassian/github-for-jira/blob/main/src/jira/client/axios.ts#L118

Reply:
Thanks @bgvozdev and @rachellerathbone , good call!
I've updated it to histogram.

Reply:
Oh and thanks for the code sample. I didn't use the bucket/histogram however, as I don't know what the bucket should be. So just call that `histogram`  function is should generate the default (count/median/lower_xx/upper_xx), I will start from there.